### PR TITLE
Fixed styling issue arising from the removal of iui-input from uicore-autosuggest__input

### PR DESCRIPTION
**Pull request by EK723**
When iTwinUI v2's `iui-input` was removed from `uicore-autosuggest__input`, a styling issue arose for the input of AutoSuggest. This PR fixes that problem.

### CODE REVIEW

**Comment by jjbeckman13** [comment]
Package versions for AppUi were bumped to ^4.10.0 because the changes in this PR are needed since AppUi uses iTwinUi v3 starting with 4.10.x

**Comment by jjbeckman13** [comment]
Before:
image
After:
image

**Review by aruniverse — APPROVED** [review]
this should be ok

File: packages/itwin/geo-tools/package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "@itwin/core-markup": "^4.0.0",
     "@itwin/core-orbitgt": "^4.0.0",
     "@itwin/core-quantity": "^4.0.0",
-    "@itwin/core-react": "^4.0.1",
+    "@itwin/core-react": "^4.10.0",
     "@itwin/core-telemetry": "^4.0.0",
     "@itwin/imodel-components-react": "^4.0.1",
```
Comment (aruniverse) [inline packages/itwin/geo-tools/package.json:61]: shouldnt imodel-components-react be updated with the rest of the appui deps?

File: packages/itwin/geo-tools/package.json [hunk 2]
```diff
@@ -45,9 +45,9 @@
   "devDependencies": {
     "@itwin/appui-abstract": "^4.0.0",
     "@itwin/appui-layout-react": "^4.0.1",
```
Comment (aruniverse) [inline packages/itwin/geo-tools/package.json:47]: i think if you are updating appui peers, you can drop appui-layout-react

**Review by aruniverse — APPROVED** [review]

**Review by mdastous-bentley — APPROVED** [review]
looks good in test-viewer.

**Pull request merged**